app/static/styles/main.css

```diff
@@ -144,4 +151,35 @@ body.resume header div div a.resume, body.contact header div div a.contact {
     color: white;
     font-size: 50px;
     font-weight: 700;
+}
+
+
+/* for About Page */
+
+.about {
+    background-color: #56bfc9;
+}
+
+.about img {
+    width: 100%;
+    height: 100%;
+    background-color: #d9d9d9;
+    border-radius: 50px;
+}
+
+.about table {
+    padding-left: 5%;
+    padding-top: 2rem;
+    padding-bottom: 1rem;
```

Comment:
I like symmetry so the different padding on top and bottom bothers me a little haha

Reply:
tried a different method, looks good so far